extensions/sql-migration/src/wizard/loginSelectorPage.ts

```diff
@@ -255,24 +276,120 @@ export class LoginSelectorPage extends MigrationWizardPage {
 			})
 			.component();
 
-		await this._loadLoginList();
+		this._windowsAuthInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				style: 'information',
+				text: constants.LOGIN_MIGRATIONS_SELECT_LOGINS_WINDOWS_AUTH_WARNING,
+				CSSStyles: { ...styles.BODY_CSS, 'display': 'none', }
+			}).component();
+
+		await this._createSystemLoginTablesTab(view);
+		await this._createNonSystemLoginTablesTab(view);
+
+		this._tabs = view.modelBuilder.tabbedPanel()
+			.withTabs([this._nonSystemloginTablesTab, this._systemLoginTablesTab])
+			.withProps({
+				CSSStyles: { 'margin-top': '8px', }
+			})
+			.component();
+
+		const flex = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			height: '100%',
+		}).withProps({
+			CSSStyles: {
+				'margin': '-20px 28px 0px 28px'
+			}
+		}).component();
+		flex.addItem(this._windowsAuthInfoBox, { flex: '0 0 auto' });
```

Comment:
may be define flex { flex: '0 0 auto' } as  variable since it is being used multiple times.

https://github.com/microsoft/azuredatastudio/blob/abb643c27729e1c7675156c6468990182de32cf5/extensions/arc/src/ui/components/keyValueContainer.ts#L47